core/i18n/resources/en.js

```diff
@@ -73,6 +73,9 @@ Do you want to proceed?`,
     measurePrevSteps: 'Measure Previous Steps',
     measurePrevSteps_plural: 'Measures Previous Steps',
     min: 'Minimum',
+    mr: 'Mr',
+    ms: 'Ms',
+    preferNotToSay: 'Prefer not to say',
```

Comment:
maybe this should go into a userTitle property?

userProps.title =  {mr, ms, p.. }